store.go

```diff
@@ -3187,9 +3188,15 @@ func stringSliceWithoutValue(slice []string, value string) []string {
 	modified := make([]string, 0, len(slice))
```

Comment:
Doing a non-trivial edit like this in a function innocuously named `stringSliceWithoutValue` is horribly confusing and almost certain to trip some future caller up. If this needs to exist at all, rename it to be clear about what the function does.